src/app/dso-shared/dso-edit-metadata/metadata-field-selector/metadata-field-selector.component.ts

```diff
@@ -112,32 +143,33 @@ export class MetadataFieldSelectorComponent implements OnInit, OnDestroy, AfterV
    * Update the mdFieldOptions$ depending on the query$ fired by querying the server
    */
   ngOnInit(): void {
+    this.input.valueChanges.pipe(
+      debounceTime(this.debounceTime),
+      startWith(''),
+      tap( () => this.currentPage$.next(1) )
+    ).subscribe((valueChange) => {
```

Comment:
Please also add this subscribe to the subs array, so it can be unsubscribed onDestroy

Reply:
@artlowel  I aded to subs array